adapters/rtbhouse/rtbhouse.go

```diff
@@ -166,3 +187,34 @@ func (adapter *RTBHouseAdapter) MakeBids(
 	return bidderResponse, nil
 
 }
+
+func getMediaTypeForBid(bid openrtb2.Bid) openrtb_ext.BidType {
+	switch bid.MType {
+	case openrtb2.MarkupBanner:
+		return openrtb_ext.BidTypeBanner
+	case openrtb2.MarkupNative:
+		return openrtb_ext.BidTypeNative
+	default:
+		return openrtb_ext.BidTypeBanner
```

Comment:
should not default to banner type. Should return error if found an unknown bid type

Reply:
@piotrj-rtbh Thanks for the clarifiacation

If bidder skips setting `mtype` then can fallback to check if imp.Banner is not null and then return  `openrtb_ext.BidTypeBanner`

Reply:
> If bidder skips setting `mtype` then can fallback to check if imp.Banner is not null and then return `openrtb_ext.BidTypeBanner`
@onkarvhanumante I didn't get it well. Do you mean I need to loop through request's `imp[]` array, find matching element (by `impid` I suppose) and check it's `Banner` prop if it's null or not?

Reply:
yes

> > If bidder skips setting `mtype` then can fallback to check if imp.Banner is not null and then return `openrtb_ext.BidTypeBanner`
> 
> @onkarvhanumante I didn't get it well. Do you mean I need to loop through request's `imp[]` array, find matching element (by `impid` I suppose) and check it's `Banner` prop if it's null or not?

yes for default case to check if banner was set in `req.Imp[]`. However this may not work if your adapter support multi-format imp

Reply:
I think if you wanna default to Banner then you could use the following code to check the native first and then default to banner. 
```
func getMediaTypeForBid(bid openrtb2.Bid) openrtb_ext.BidType {
	switch bid.MType {
	case openrtb2.MarkupNative:
		return openrtb_ext.BidTypeNative
	default:
		return openrtb_ext.BidTypeBanner
}
```

The above should work for your adapter. But I strongly recommend that you should change your bidder to properly set MType and not skip that value.

Reply:
@onkarvhanumante @gargcreation1992 

We decided to always return `MType` from the bidder. Just in case the bid type detecting function now may return an error if `MType` is not supported. Returned bids which have unknown `MType` are skipped then.